homeassistant/components/zwave_js/select.py

```diff
@@ -40,6 +42,38 @@ def async_add_select(info: ZwaveDiscoveryInfo) -> None:
     )
 
 
+class ZwaveSelectEntity(ZWaveBaseEntity, SelectEntity):
+    """Representation of a Z-Wave select entity."""
+
+    def __init__(
+        self, config_entry: ConfigEntry, client: ZwaveClient, info: ZwaveDiscoveryInfo
+    ) -> None:
+        """Initialize a ZwaveSelectEntity entity."""
+        super().__init__(config_entry, client, info)
+
+        # Entity class attributes
+        self._attr_name = self.generate_name(include_value_name=True)
+        self._attr_options = list(self.info.primary_value.metadata.states.values())
+
+    @property
+    def current_option(self) -> str | None:
+        """Return the selected entity option to represent the entity state."""
+        return str(
+            self.info.primary_value.metadata.states.get(
+                str(self.info.primary_value.value), self.info.primary_value.value
```

Comment:
If `self.info.primary_value.value` is None we may return a string `"None"` here. Do we need a guard against `self.info.primary_value.value` being None?